components/table/table-col-sort-button.js

```diff
@@ -107,11 +117,20 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 		`;
 	}
 
+	static #columnDataTypeKeys = {
+		words: ['components.table.a-to-z', 'components.table.z-to-a'],
+		numbers: ['components.table.low-to-high', 'components.table.high-to-low'],
+		dates: ['components.table.old-to-new', 'components.table.new-to-old']
+	};
+
+	static #buttonDescription = ['components.table.change-sort-order', 'components.table.add-sort-order'];
```

Comment:
It might be clearer to entirely use objects instead of arrays for these. For example:
```
static #columnDataTypeKeys = {
words: { asc: 'components.table.a-to-z', desc: 'components.table.z-to-a' }
};
```
etc.